src/V8JsiRuntime_impl.h

```diff
@@ -640,11 +644,38 @@ class V8Runtime : public facebook::jsi::Runtime {
   static void GCEpilogueCallback(v8::Isolate *isolate, v8::GCType type, v8::GCCallbackFlags flags);
 
  private:
+ // RAII wrappers for multi threaded support - order of the various scopes matters
+  struct IsolateScope {
+    IsolateScope(const V8Runtime *runtime) : 
```

Comment:
> * [](http://example.com/codeflow?start=32&length=2)

nit: This and IsolateLocker can take "const V8Runtime &"

Reply:
But all the callsites would need to pass `*this` which is ugly :)

Reply:
:smile: